BoxContentSDK/BoxContentSDK/Requests/BOXFileRepresentationDownloadRequest.h

```diff
@@ -12,17 +12,59 @@
 
 @interface BOXFileRepresentationDownloadRequest : BOXRequestWithSharedLinkHeader
 
+// This is not the etag of a particular version of the file, nor the sequential version number,
+// it is the ID of the version representation gotten from /files/<fileID>/versions
 @property (nonatomic, readwrite, strong) NSString *versionID;
+
+// Enable NSURLSession cachepolicy for this request
 @property (nonatomic, readwrite, assign) BOOL ignoreLocalURLRequestCache;
 
+/**
+ Request will download file into destinationPath, and the file download can continue
+ running in the background even if app is not running.
+ 
+ @param destinationPath File path to download file.
+ @param fileID BOXFile modelID
+ @param representation BOXRepresentation of available content to download. See BOXFileRequestInfo to
+                       retrieve available representation.
+ @return instance of BOXFileRepresentationDownloadRequest
+ */
 - (instancetype)initWithLocalDestination:(NSString *)destinationPath
                                   fileID:(NSString *)fileID
                           representation:(BOXRepresentation *)representation;
 
+/**
+ Similar to the above init method, request will download file into destinationPath,
+ and the file download can continue running in the background even if app is not running
+ If downloadTask is provided, it will be used to execute/reconnect with the existing download task
```

Comment:
if associateID is provided,...

Could we update equivalent comment for BOXFileDownloadRequest as well?

Reply:
done